pkg/proxy/proxy.go

```diff
@@ -76,8 +76,16 @@ func ConvertHostsToRedirectURLs(hosts []string, sso *apiv1.SSO) []string {
 	return redirectURLs
 }
 
-func buildName(name string, namespace string) string {
-	return fmt.Sprintf("%s-%s", namespace, name)
+func buildName(resourceName string, suffix string) string {
```

Comment:
This function is now more generic and can is aimed to handle any suffix (not only namespaces)

Reply:
Looks good! Do you mind adding a small unit test for `buildName`? Thanks